### PR TITLE
Add documentation for live patching systemVMs

### PR DESCRIPTION
**Pull request by Pearl1594**
Doc PR for : https://github.com/apache/cloudstack/pull/5831

### CODE REVIEW

**Comment by Pearl1594** [comment]
@blueorangutan docbuild

**Comment by Pearl1594** [comment]
@rohityadavcloud Could you please help review the content of the doc PR. Thanks

File: source/upgrading/upgrade/_sysvm_restart.rst [hunk 1]
```diff
@@ -15,7 +15,22 @@
...
 
-Once you've upgraded the packages on your management servers, you'll
+.. note::
+
+   From Apache CloudStack version 4.17.0 onward, there is support to live patch 
+   systemVMs, namely, SSVM, CPVM, Routers. Live patching provides support 
+   for zero-downtime upgrades, wherein, the SystemVM software is updated to the
+   latest code version without having to destroy and recreate them / restart them.
+
+   With this feature, users will have a choice wherein they can use the existing systemVM template with the latest
```
Comment (rohityadavcloud) [inline source/upgrading/upgrade/_sysvm_restart.rst:25]:
```suggestion
   With this feature, users will have a choice wherein they can use the existing System VM template with the latest
```

Comment (rohityadavcloud) [reply source/upgrading/upgrade/_sysvm_restart.rst:25]: (comment being, use consist terms - either systemvm or SystemVM or System VM or system-vm

File: source/upgrading/upgrade/_sysvm_restart.rst [hunk 2]
```diff
@@ -15,7 +15,22 @@
...
+   software by using the live patch feature, or can follow the usual workflow of restarting the
+   systemVM to use the latest systemVM template. Live Patching systemVMs serves to be especially
+   useful in cases when the code version has upgraded but the template hasn't. In such a scenario users
+   will no longer need to restart the systemVMs to use the latest code.
```
Comment (rohityadavcloud) [inline source/upgrading/upgrade/_sysvm_restart.rst:29]: @Pearl1594 can you explain in some more detail what is covered in the "zero downtime" and what isn't, esp in the context of CPVM and VRs.

**Review by rohityadavcloud — CHANGES_REQUESTED** [review]
Left some remarks, but LGTM. Need a bit of polish, may consult with Giles/Ivet - basically we should be consistent in the docs as we're in marketing/blog etc.

**Comment by Pearl1594** [comment]
@blueorangutan docbuild

**Comment by nvazquez** [comment]
@blueorangutan docbuild

File: source/upgrading/upgrade/_sysvm_restart.rst [hunk 3]
```diff
@@ -15,9 +15,46 @@
 
 .. sub-section included in upgrade notes.
 
-Once you've upgraded the packages on your management servers, you'll
+.. note::
```
Comment (nvazquez) [inline source/upgrading/upgrade/_sysvm_restart.rst:18]: This is now very long for a note :)

File: source/upgrading/upgrade/_sysvm_restart.rst [hunk 4]
```diff
@@ -15,9 +15,46 @@
...
+            +-----------------+---------------------------+
+            | CPVM            | cloud                     |
+            +-----------------+---------------------------+
+            | VRs             | haproxy, apache2, dnsmasq |
+            +-----------------+---------------------------+
+
+   Also, with respect to VRs, a network restart without cleanup is initiated to ensure all rules
+   are re-applied. 
+
+   **NOTE:** In case there is an absolute need to upgrade the system VM template due to availability of
```
Comment (nvazquez) [inline source/upgrading/upgrade/_sysvm_restart.rst:50]: Could apply ..note in this case

**Review by nvazquez — APPROVED** [review]

**Comment by Pearl1594** [comment]
@blueorangutan docbuild

**Pull request merged**